legal/geschaeftsordnung.md

```diff
@@ -0,0 +1,27 @@
+# Geschäftsordnung
+
+## Aufgaben der Organe des Vorstandes
+
+1. Sprecher*in
+
+    Aufgabenbeschreibung.
```

Comment:
```suggestion
    Die Sprecher*in koordiniert die kommunikative Arbeit des Vereins.
```

Reply:
Kommentar CK: Vertritt den Verein nach außen, ggf. nach Abstimmugn mit Vorstand

Reply:
Vorschlag aus meiner Sicht in Ordnung.